mlrun/runtimes/function.py

```diff
@@ -1021,7 +1036,12 @@ def get_nuclio_deploy_status(
         verbose,
         resolve_address,
         return_function_status=True,
+        auth_info=_to_nuclio_auth_info(auth_info),
     )
 
     text = "\n".join(outputs) if outputs else ""
     return state, address, name, last_log_timestamp, text, function_status
+
+
+def _to_nuclio_auth_info(auth_info: AuthInfo = None):
```

Comment:
Why not making it a method of `AuthInfo` class ?

Reply:
done